app/views/document_images/crop.html.erb

```diff
@@ -1,25 +1,26 @@
 <% content_for :back_link, document_images_path(@document) %>
 <% content_for :title, t("document_images.crop.title", title: @document.title_or_fallback) %>
 
-<p class="govuk-body">
-  <%= t("document_images.crop.description") %>
-</p>
-
 <%= form_tag(crop_document_image_path(@document, @image), method: :patch) do %>
   <%= hidden_field_tag :wizard, params[:wizard] %>
 
   <%= render "/components/image_cropper", {
     id: "crop-image",
+    description: t("document_images.crop.description"),
     src: url_for(@image.blob),
+    no_js_description: t("document_images.crop.no_js_description"),
+    no_js_src: url_for(@image.crop_variant('960x640')),
+    exact_dimensions: @image.at_exact_dimensions?,
+    exact_dimensions_description: t("document_images.crop.exact_dimensions_description"),
     alt_text: @image.alt_text,
     crop_x: @image.crop_x,
     crop_y: @image.crop_y,
     crop_width: @image.crop_width,
-    crop_height: @image.crop_height
+    crop_height: @image.crop_height,
   } %>
 
   <%= render "govuk_publishing_components/components/button", {
-    text: "Crop image",
+    text: (@image.at_exact_dimensions?) ? "Continue" : "Crop image",
```

Comment:
I think we also have a weirdness here: if JavaScript is disabled, we still say 'Crop image', even though the description above says it has already been 'automatically resized to fit'. Perhaps one for @benhazell :-).